api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/core/layer/convolutional/SeparableConv2D.kt

```diff
@@ -119,45 +105,59 @@ public class SeparableConv2D(
         fanIn = (inputDepth * kernelSize[0] * kernelSize[1]).toInt()
         fanOut = ((outputDepth * kernelSize[0] * kernelSize[1] / (strides[0].toDouble() * strides[1])).roundToInt())
 
-        createSeparableConv2DVariables(tf, kGraph)
+        createSeparableConv2DVariables(tf, kGraph, numberOfChannels)
```

Comment:
I would rather inline this method here. It seems to me that all the previous steps, like computing `numberOfChannels`, `inputDepth`/`outputDepth`, `fanIn`/`fanOut` are also the part of creating variables.